core/src/ast.rs

```diff
@@ -774,7 +810,7 @@ pub(crate) fn resolve_identifier<I: Interrupt>(
 		"tomorrow" => Value::Date(crate::date::Date::today(context)?.next()),
 		"yesterday" => Value::Date(crate::date::Date::today(context)?.prev()),
 		"trans" => Value::String(Cow::Borrowed("🏳️‍⚧️")),
-		_ => return crate::units::query_unit(ident.as_str(), attrs, context, int),
+		_ => return Err(FendError::IdentifierNotFound(ident.clone())),
```

Comment:
This causes an `Cow<&'static, str>` clone every time this function is called... which is **not used** as the "unknown identifier" error is taken from `unit_result`.

Is it worth removing this unnecessary clone?

Reply:
Not sure this really matters either way :)